plugins/services/src/js/containers/pod-debug/PodDebugContainer.js

```diff
@@ -207,30 +208,54 @@ class PodDebugTabView extends React.Component {
       return null;
     }
 
+    let message, primaryAction;
+
+    if (this.props.pod.isDelayed()) {
```

Comment:
The method name is now misleading, Delayed means not that it is waiting for resources.
I would suggest renaming the method to `getUnableToStartNotice`

Reply:
Do you mean the "isDelayed" method? It is not part of this PR, but I can create another PR to rename it.